src/views/Map/Map.tsx

```
@@ -127,29 +127,29 @@ export function MapPage() {
           searchBar={
-            featureFlags[FeatureFlag.SEARCH_BAR] &&
-            featureFlags[FeatureFlag.SEARCH_LOCATION] && <Search />
+            featureFlags[AppFeature.SEARCH_BAR] &&
+            featureFlags[AppFeature.SEARCH_LOCATION] && <Search />
           }
           analytics={<Analytics />}
           // if EVENTS_LIST is enabled, we always have default feed
-          disasters={featureFlags[FeatureFlag.EVENTS_LIST] && <EventListPanel />}
+          disasters={featureFlags[AppFeature.EVENTS_LIST] && <EventListPanel />}
           layersAndLegends={<LayersAndLegends />}
           matrix={<></>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)